Fhi.HelseId.Samples/HostBuilders/TestHostBuilderSamples.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Net;
+using Fhi.HelseId.Web.ExtensionMethods;
+using Fhi.TestFramework.Extensions;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using NUnit.Framework;
+
+namespace Fhi.Samples.TestServerSetup
+{
+    /// <summary>
+    /// Using HostBuilder with TestServer
+    /// </summary>
+    public class TestServerSamples
+    {
+        [Test]
+        public async Task UsingHostBuilder()
+        {
+            IConfigurationRoot inMemoryConfig = CreateConfig();
```

Comment:
Nit: Is there a reason for not using "var" here?

This is repeated several places.

Reply:
No. VS fault when using refactoring tooling.. ;)